src/android/toga_android/widgets/detailedlist.py

```diff
@@ -33,9 +33,14 @@ class DetailedList(Widget):
     def create(self):
         # DetailedList is not a specific widget on Android, so we build it out
         # of a few pieces.
-        parent = android_widgets.LinearLayout(self._native_activity)
-        parent.setOrientation(android_widgets.LinearLayout.VERTICAL)
-        self.native = parent
+
+        if self.native is None:
+            self.native = android_widgets.LinearLayout(self._native_activity)
+            self.native.setOrientation(android_widgets.LinearLayout.VERTICAL)
+        else:
+            # If create() is called a second time, clear the widget and regenerate it.
+            self.native.removeAllViews()
```

Comment:
In what context does this happen? I can't think of a context where create should be called twice...

Reply:
There's two reasons I want to support calling `create()` twice. One is aesthetic, and the other is the practical issue that made me submit the pull request.

The first is an aesthetic consider about public methods. It seems a little cruel that if someone _does_ call this method twice, the entire app crashes with a JNI error.

The second is that, at the moment, DetailedList on Android regenerates the entire widget when any new data is added to the list. It does so by calling `.create()`! This is the reason that clicking the "Insert Row" button causes the app to crash.

Reply:
Ok - I think I remember this coming up in the original review. It's not ideal (I'd rather not have the content completely recreated when you add a single row), but that's not a new problem.